src/components/components/Badge/Badge.scss

```diff
@@ -1,4 +1,4 @@
-@import "../../../frontend-elements/styles/main";
+@import '../../../frontend-elements/styles/main';
```

Comment:
Why not use the magic paths like other imports?

Reply:
done